features/breadcrumbs.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Breadcrumbs
+  In order to use the skip link,
+  As a user
+  I want the breadcrumbs to be outside the main content
+
+  Scenario: When there are breadcrumbs on the page
+    Given the page exists
```

Comment:
Given there exists (something here) content item

Reply:
Or instead of doing all of that you can reuse an existing step:

    When I view the research and statistics finder with a topic param set